indigo_app/templates/indigo_app/place/_work_detail.html

```diff
@@ -84,3 +84,33 @@
     <div class="tab-pane" id="work-{{ work.pk }}-tasks"></div>
   {% endif %}
 </div>
+
+{% if perms.indigo_api.change_work or perms.indigo_api.bulk_add_work %}
+  <div class="border-top p-2 {% if work.work_in_progress %}bg-work-in-progress-light{% else %}bg-light{% endif %}">
+    {% if perms.indigo_api.change_work %}
+      <button
+          id="edit_work_button"
```

Comment:
drop the id, it's not unique on the page and isn't necessary (I don't think anything refers to it)